impl_core/src/main/java/io/opencensus/implcore/trace/export/SpanExporterImpl.java

```diff
@@ -32,9 +38,37 @@
 import java.util.logging.Logger;
 import javax.annotation.concurrent.GuardedBy;
 
+/*>>>
+import org.checkerframework.checker.nullness.qual.Nullable;
+*/
+
 /** Implementation of the {@link SpanExporter}. */
 public final class SpanExporterImpl extends SpanExporter {
   private static final Logger logger = Logger.getLogger(ExportComponent.class.getName());
+  private static final DerivedLongCumulative droppedSpans =
+      Metrics.getMetricRegistry()
```

Comment:
Tracing impl now depends on Metrics - that seems fine - I prefer not to have cyclic dependencies but, iiuc, this doesn't introduce them. Interesting issue, tho - if we split the artifacts for tracing and metrics, this won't work.

Reply:
It depends only on the API artifact not the impl artifact.

Reply:
My second point was that a service that links in a tracing impl but not a metrics impl will get unexpected results - not an issue with the current way we package artifacts but could be a problem if we ever decide to split the implementation artifacts.